src/goalServices/goalUpdate.test.js

```diff
@@ -216,8 +217,5 @@ describe('Change Goal Status', () => {
     expect(updatedGoals.length).toEqual(1);
     const updatedGoal = updatedGoals[0];
     expect(updatedGoal.status).toEqual(newStatus);
-    expect(updatedGoal.closeSuspendReason).toEqual(reason);
-    expect(updatedGoal.closeSuspendContext).toEqual(context);
-    expect(updatedGoal.previousStatus).toEqual(oldStatus);
```

Comment:
Removed these assertions because closeSuspendReason and closeSuspendContext are *not* going to be stored on the `Goal` any more.